### DISCUSSION TITLE
Add cvar to disable :setMaterial() being used player ents

### DISCUSSION THREAD

**Discussion by Isemenuk27** [wiremod/wire#2933] [Suggestions]
Add cvar to disable :setMaterial() being used player ents

**Comment by vurvdev** [comment]
That's on your prop protection to do tho? It checks ownership:

https://github.com/wiremod/wire/blob/master/lua/entities/gmod_wire_expression2/core/entity.lua#L508

**Reply by Denneisk** [reply]
I think he means using it on the owner player. Some materials let you go invisible so I guess that's the idea here.

**Reply by vurvdev** [reply]
> I think he means using it on the owner player. Some materials let you go invisible so I guess that's the idea here.

That's still up to your prop protection to not give perms on your own player
